javascriptv3/example_code/iam/src/iam_listpolicies.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/iam-examples-managing-users.html.
+Purpose:
+iam_listpolicies.js demonstrates how to list list all the managed policies that are available in your
```

Comment:
iam_listpolicies.js demonstrates how to list all the managed policies that are available in your
AWS account. This includes your own customer-defined managed policies and all AWS
managed policies.

Reply:
done